### DISCUSSION TITLE
Any news on x-teleport support?

### DISCUSSION THREAD

**Discussion by TixieSalander** [livewire/livewire#5184] [4. General]
Hi!
As a Livewire and AlpineJS user It's been a while since I'm waiting for AlpineJS's x-teleport support by Livewire. 
I need it to make simple modal elements (who require to be moved outside of their stacking-content to not be overflowed) inside of livewire element or that have Livewire element inside its content. 

But i can imagine how tricky it may be for Livewire to figuring things out when a JS tool is yeeting DOM at the other side of the document.

Support is [apparently on the roadmap](https://alpinejs.dev/directives/teleport), but does anyone have actual infos about that?

✌️

**Comment by SebastiaanKloos** [comment]
I will definitely sponsor Livewire when this is being implemented. I need this so much to be able to use nested `<form>` components

**Comment by TixieSalander** [comment]
Looks this problem gonna be resolved by this addition of a teleport directive in the next livewire version! 🎉

![3FB7FFBF-9746-4133-BADE-809A54021A64](https://user-images.githubusercontent.com/2143796/190472691-c9f938f5-0f08-45cd-b2e1-e2a2be879ad8.jpeg)

(Source: https://twitter.com/freezydorito/status/1570404094719791104)

**Comment by michael-rubel** [comment]
The teleport feature will be available in Livewire v3. 🙂 
@calebporzio presented it at [Laracon Online Summer](https://www.youtube.com/watch?v=f4QShF42c6E&t=6014s)
